package.json

```diff
@@ -14,6 +14,7 @@
   "dependencies": {
     "@changesets/changelog-github": "^0.4.2",
     "@changesets/cli": "^2.18.0",
+    "@sentry/node": "^6.16.1",
```

Comment:
why did we add this to the root package.json? it's also in packages/wrangler/package.json

Reply:
Unintentional. I will make a PR to remove